### PR TITLE
Add a command class for each command available in MenuParser.

### LINKED ISSUE #54: Add specific command classes
for each command available in v1.0 of Eduk8.

### PR DESCRIPTION
**Pull request by josephhhhhhhhh**
Closes #54.

### CODE REVIEW

File: src/main/java/seedu/eduke8/command/Command.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 public abstract class Command {
-    private boolean isExit;
+    protected boolean isExit;
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/command/Command.java:8]: Nice catch on the accessibility

File: src/main/java/seedu/eduke8/parser/MenuParser.java [hunk 2]
```diff
@@ -26,34 +30,21 @@ public Command parseCommand(DisplayableList topicList, String userInput) {
...
         case "quiz":
-            parseQuizCommand((TopicList) topicList, numOfQuestions, topicName, ui);
-            break;
+            return new QuizCommand((TopicList) topicList, numOfQuestions, topicName, ui);
         case "exit":
-            System.exit(0);
-            break;
+            return new ExitCommand();
         default:
             break;
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/parser/MenuParser.java:48]: Maybe we could add something here for users who had wrote commands not in our list. I can create something in Ui

**Review by anqi20 — APPROVED** [review]
Looking pretty good!

File: text-ui-test/input.txt [hunk 3]
```diff
@@ -1 +1 @@
-bye
\ No newline at end of file
+exit
```
Comment (kstonekuan) [inline text-ui-test/input.txt:4]: Good update! Did you managed to test the other commands with IO testing also?

File: src/main/java/seedu/eduke8/command/IncorrectCommand.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    String errorType = "";
+
+    public IncorrectCommand(String errorType) {
+        this.errorType = errorType;
+    }
+
+    @Override
+    public void execute(DisplayableList displayableList, Ui ui) {
+        ui.printError(); //to implement in ui: print out messages according to string input indicating type of error
+    }
```
Comment (Zhi-You) [inline src/main/java/seedu/eduke8/command/IncorrectCommand.java:16]: Remember to remind @anqi20 about this again @josephhhhhhhhh !

File: text-ui-test/EXPECTED.TXT [hunk 5]
```diff
@@ -10,5 +10,23 @@ Hello! I'm E-Duke-8
...
+3) topics
+4) textbook
+5) quiz
+6) stats
+7) exit
+-------------------------------------------------------------------
+-------------------------------------------------------------------
+The textbook for this module is available at:  https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/index.html
+-------------------------------------------------------------------
+-------------------------------------------------------------------
```
Comment (Zhi-You) [inline text-ui-test/EXPECTED.TXT:30]: Great! Looks like a working product is right on the way!

File: src/main/java/seedu/eduke8/command/IncorrectCommand.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
+package seedu.eduke8.command;
+
+import seedu.eduke8.common.DisplayableList;
+import seedu.eduke8.ui.Ui;
+
+public class IncorrectCommand extends Command {
```
Comment (Zhi-You) [inline src/main/java/seedu/eduke8/command/IncorrectCommand.java:6]: Logic seems great for this class!

**Review by Zhi-You — APPROVED** [review]
LGTM!

**Pull request merged**